docs/getting_started.md

```diff
@@ -3,42 +3,48 @@
 
 ## Prerequisites
 
-1. Python >= 3.6
-2. Configure AWS Account credentials when testing with AWS platforms,
-   check default section in `~/.aws/credentials` for access/secret key
-   [check aws-configuration](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
-3. oc client binary is installed on your localhost and binary is listed in $PATH
-   (running oc version on terminal should display version > 3.11).
-   Latest client can be downloaded from [oc-client](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/).
-4. For vSphere based installations, [terraform](https://learn.hashicorp.com/terraform/getting-started/install.html)
-   and [jq]( https://stedolan.github.io/jq/download/) should be installed ( terraform version should be 0.11.13  )
-5. For UI testing follow the instructions in [openshift console](https://github.com/openshift/console#dependencies) repository.
-   After clone of repository and setting all dependencies please create configuration file with RUN section with `openshift_console_path`
-   parameter pointing to your local clone of openshift console repo. See example in `conf/ocsci/ui-testing.yaml`.
+* Python >= 3.7
+* oc client binary available in the $PATH.
```

Comment:
I'm not sure if this is true anymore as we download the `oc` binary into `ocs-ci/bin` and use that.

Reply:
Interesting. As I progress running the tools/tests I'll try it without 'oc' on my path to see if that's the case.

Reply:
@ebenahar @phlogistonjohn this is done as a part of our [cluster fixture](https://github.com/red-hat-storage/ocs-ci/blob/master/tests/conftest.py#L1061) so the user shouldn't need to do this as a part of prereqs

Reply:
I agree this should not be part of fresh deployment, for existing cluster we may be missing some code fix but in general this should not be prereq.